clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -254,12 +264,50 @@ def center_nifti_origin(input_image, output_image):
             qform[i - 1, 3] = -1.0 * hd['pixdim'][i] * hd['dim'][i] / 2.0
         new_img = nib.Nifti1Image(canonical_img.get_data(caching='unchanged'), qform)
 
-    except (OSError, nib.orientations.OrientationError) as e:
-        return None
+        nib.save(new_img, output_image)
+        if not isfile(output_image):
+            error_str = Fore.RED + '[Error] Nifti created but clinica could not save it to ' \
+                        + output_image + '. Please check that the output folder has the correct permissions.' \
+                        + Fore.RESET
+    return output_image, error_str
```

Comment:
Some typo suggestions (Nifti -> NIfTI, clinica, Clinica)
```suggestion
            error_str = Fore.RED + '[Error] NIfTI created but Clinica could not save it to ' \
                        + output_image + '. Please check that the output folder has the correct permissions.' \
                        + Fore.RESET
```